src/components/AdjustableSidebarWidth.vue

```diff
@@ -113,6 +114,14 @@ export default {
       type: Boolean,
       default: false,
     },
+    disableResizing: {
```

Comment:
Does this need to be a separate prop? I was thinking that it should probably be implied that the component is not meant to be resizable if you're already setting an explicit width for it.

Reply:
I thought about it more and probably no...